docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -118,3 +118,73 @@ For more information, refer to the [Converting a Kaldi Model](prepare_model/conv
 - To get conversion recipes for specific TensorFlow, ONNX, PyTorch, Apache MXNet, and Kaldi models,
 refer to the [Model Conversion Tutorials](prepare_model/convert_model/Convert_Model_Tutorials.md).
 - For more information about IR, see [Deep Learning Network Intermediate Representation and Operation Sets in OpenVINO™](IR_and_opsets.md).
+
+## MO Python API
+
+- Model Optimizer has Python API for model conversion, which is represented by convert_model() method in openvino.tools.mo namespace.
+  convert_model() has all the functionality available from command line tool plus the possibility of passing Python objects (models, extensions) to convert_model() directly from memory.
+  convert_model() returns openvino.runtime.Model object which can be compiled and infered or serialized to IR.
+
+```sh
+from openvino.tools.mo import convert_model
+
+ov_model = convert_model("resnet.onnx")
+```
+
+MO Python API allows conversion of PyTorch models. Converting PyTorch models requires providing "input_shape" or "example_input".
+```sh
+import torchvision
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, input_shape=[1,3,100,100])
+```
```

Comment:
Propose to leave only one most popular example of usage of Python API, for example, PyTorch. For other supported types please refer to FW specific documents.
Here we should have some general information about Model Optimizer Python API: supported objects, API details, and one example.

Reply:
Done.